doc/kubernetes/modules/ROOT/pages/running/split-brain-stonith.adoc

```diff
@@ -0,0 +1,124 @@
+= Deploy an AWS Lambda to guard against Split-Brain
+:description: This guide explains how to reduce the impact when split-brain scenarios occur between two sites in an Active/Active deployment.
+
+{description}
+
+== Architecture
+In the event of a network communication failure between the two sites in a Multi-AZ Active/Active deployment, it is no
+longer possible for the two sites to continue to replicate session state between themselves and the two sites
+will become increasingly out-of-sync. As it's possible for subsequent Keycloak requests to be routed to different
+sites, this may lead to unexpected behaviour as previous updates will not have been applied to both sites.
+
+Typically in such scenarios a quorum is used to determine which sites are marked as online or offline, however as Active/Active
+deployments only consist of two sites, this is not possible. Instead, we leverage the STONITH (Shoot The Other In The Head)
+pattern to ensure that once a split-brain has been detected, only one site can continue to serve user requests.
+
+STONITH is implemented by using https://prometheus.io/docs/alerting/latest/overview/[Prometheus Alerts] to call an AWS Lambda
+based webhook whenever a site's Infinispan cluster detects that the other site is no longer in the JGroups site view. The
```

Comment:
I probably rewrite this sentence without using `JGroups site view`, otherwise users may be wondering what it is.

My 2 cents: `to call an AWS Lambda based webhook whenever one of the sites is unable to establish or connect to the other side.`